packages/finicity-adapter/src/adapter.test.ts

```diff
@@ -323,7 +409,7 @@ describe("finicity aggregator", () => {
       );
     });
 
-    it("handles 'error' reason with code 201 and updates connection status to FAILED", async () => {
+    it("handles 'error' reason with code 201 and updates connection status to Connected", async () => {
```

Comment:
It turns out code 201 means the widget exited only because there was nothing to do because the connection is already good. This is the response that comes back when you try to refresh a finicity connection which doesn't need refreshing.